torch_frame/config/text_tokenizer.py

```diff
@@ -0,0 +1,22 @@
+from dataclasses import dataclass
+from typing import Callable, List, Optional
+
+from torch_frame.typing import TensorData
+
+
+@dataclass
+class TextTokenizerConfig:
+    r"""Text tokenizer that maps a list of strings/sentences into a
+    dictionary of :class:`MultiNestedTensor`.
+
+    Args:
+        text_tokenizer (callable): A callable text tokenizer that takes a list
+            of strings as input and output the dictionary of
+            :class:`MultiNestedTensor`.
+        batch_size (int, optional): Batch size to use when encoding the
+            sentences. If set to :obj:`None`, the text embeddings will
+            be obtained in a full-batch manner. (default: :obj:`None`)
+
+    """
+    text_tokenizer: Callable[[List[str]], TensorData]
```

Comment:
The text tokenizer should output `List[Dict[str, Tensor]]`, which we then convert into `Dict[str, MultiNestedTensor]` internally.

Reply:
You can see how BERT and other standard tokenizers output. Ideally, our package can use a variety of them as they are, rather than asking the users to handle the complexity with `MultiNestedTensor`.

Reply:
Sure, will change to List[Dict[str, Tensor]]